torchci/components/metrics/panels/TimeSeriesPanel.tsx

```diff
@@ -25,6 +25,61 @@ export function getTooltipMarker(color: string) {
   );
 }
 
+export function seriesWithInterpolatedTimes(
+  data: any,
+  granularity: any,
```

Comment:
better to type these things unless it's super hard.

Reply:
i cant figure out what the types of either of these two are :((